web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/AdvancedSearchFormController.java

```diff
@@ -1,54 +1,64 @@
 package edu.unc.lib.boxc.web.access.controllers;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import edu.unc.lib.boxc.auth.api.models.AccessGroupSet;
 import edu.unc.lib.boxc.auth.fcrepo.services.GroupsThreadStore;
 import edu.unc.lib.boxc.search.api.ContentCategory;
 import edu.unc.lib.boxc.search.api.requests.SearchState;
 import edu.unc.lib.boxc.search.solr.responses.SearchResultResponse;
 import edu.unc.lib.boxc.search.solr.utils.SearchStateUtil;
 import edu.unc.lib.boxc.web.common.controllers.AbstractErrorHandlingSearchController;
+import edu.unc.lib.boxc.web.common.utils.SerializationUtil;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
 
 import javax.servlet.http.HttpServletRequest;
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
 /**
  * Handles requests to the advanced search page, sending users to the form if there are no
  * query string parameters set, or constructing a search state and sending the user to
  * get results if they have populated the form.
  * @author bbpennel
  */
 @Controller
-@RequestMapping("/advancedSearch")
 public class AdvancedSearchFormController extends AbstractErrorHandlingSearchController {
-    private List<String> FORMAT_VALUES = Arrays.stream(ContentCategory.values())
+    private final List<String> FORMAT_VALUES = Arrays.stream(ContentCategory.values())
             .map(ContentCategory::getDisplayName)
             .sorted()
             .collect(Collectors.toList());
 
-    @RequestMapping(method = RequestMethod.GET)
+    @RequestMapping(path = "/collections", method = RequestMethod.GET, produces = APPLICATION_JSON_VALUE)
```

Comment:
It is a bit confusing to have the AdvancedSearchFormController defining a `/collections` path, when there is an already existing `/collections` path in the SearchActionController:
https://github.com/UNC-Libraries/box-c/blob/main/web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/SearchActionController.java#L121
which returns admin units, while this one returns collections, so we get different results from the same endpoint depending on the Accept header provided.

I'd suggest either scoping this endpoint more (`/advancedSearch/collections`) or using the existing json API (which might not work here due to result limits). We should also do the same for the `/formats` end point.

We're also returning all fields in the collection record to the user as json now but only using the id and title. It's probably fine at this point (this is basically what the admin UI is doing) since we only have around 425 collections, but we might need to address it eventually since the cdm migration is going to add a lot more collections.